### PR TITLE
Docs: Update docker image names for oss-specific Grafana image and channel

### PR DESCRIPTION
**Pull request by pkolyvas**
Our oss-specific docker namespaces are now public: 
- https://hub.docker.com/r/grafana/grafana-oss
- https://hub.docker.com/r/grafana/grafana-oss-dev

This PR updates the docker pages in our installation guide to point to the OSS-specific namespace. All previous tags and images have been mirrored in `grafana/grafana-oss` from `grafana/grafana`.

This PR requires a sanity check on the code examples, particularly around customizing docker configurations. 

I can confirm that vanilla installs of Grafana using `grafana/grafana-oss` as the docker image are working as expected.

### CODE REVIEW

File: docs/sources/installation/docker.md [hunk 1]
```diff
@@ -17,21 +17,21 @@ You can run Grafana on your own hardware or use [Grafana Cloud](https://grafana.
...
 
-`grafana/grafana:<version>`
+`grafana/grafana-oss:<version>`
 
 This is the default image. It's based on the popular [Alpine Linux project](http://alpinelinux.org), available in [the Alpine official image](https://hub.docker.com/_/alpine). Alpine Linux is much smaller than most distribution base images, and thus leads to slimmer and more secure images.
 
 This variant is highly recommended when security and final image size being as small as possible is desired. The main caveat to note is that it uses [musl libc](http://www.musl-libc.org) instead of [glibc and friends](http://www.etalabs.net/compare_libcs.html), so certain software might run into issues depending on the depth of their libc requirements. However, most software don't have an issue with this, so this variant is usually a very safe choice.
 
-> **Note:** The `grafana/grafana:<version>` image was based on [Ubuntu](https://ubuntu.com/) before version 6.4.0.
+> **Note:** The `grafana/grafana-oss:<version>` image was based on [Ubuntu](https://ubuntu.com/) before version 6.4.0.
```
Comment (achatterjee-grafana) [inline docs/sources/installation/docker.md:26]: Is this the Ubuntu version? If not, and is it the Grafana version? I am not clear after reading this. We do not need to list a note about Grafana version 6.4, as it is old information.

File: docs/sources/installation/docker.md [hunk 2]
```diff
@@ -219,7 +219,7 @@ version: '2'
 
 services:
   grafana:
-    image: grafana/grafana:5.1.0
+    image: grafana/grafana-oss:5.1.0
```
Comment (achatterjee-grafana) [inline docs/sources/installation/docker.md:222]:
```suggestion
    image: grafana/grafana-oss:8.0.0
```

**Review by achatterjee-grafana — COMMENTED** [review]
Added doc review.

File: docs/sources/installation/docker.md (hunk 1 above)
Comment (pkolyvas) [reply docs/sources/installation/docker.md]: Good point I'll remove.

Comment (pkolyvas) [reply docs/sources/installation/docker.md]: Done!

**Review by torkelo — COMMENTED** [review]
@pkolyvas think this looks great, but the current name for the enterprise docker repo is `grafana/grafana-enterprise`

File: docs/sources/installation/docker.md [hunk 3]
```diff
@@ -3,11 +3,18 @@ title = "Run Grafana Docker image"
...
 weight = 600
+
 +++
 
 # Run Grafana Docker image
 
-You can install and run Grafana using the official Docker image. It comes in two variants: Alpine and Ubuntu.
+You can install and run Grafana using the official Docker images. Our docker images come in two editions:
+
+**Grafana Enterprise**: `grafana/grafana-ent`
```
[the remark this answers is not in the document]
Comment (pkolyvas) [reply docs/sources/installation/docker.md]: Thanks for the catch. I've fixed them throughout.

**Comment by pkolyvas** [comment]
Changes addressed, languaged and image names updated.

**Review by achatterjee-grafana — APPROVED** [review]
LGTM

**Review by torkelo — APPROVED** [review]

**Pull request merged**